### PR TITLE
Document a way to close/delete AWS accounts

### PR DESCRIPTION
**Pull request by dj-maisy**
## What?
We now have a way to close AWS accounts via Terraform. We should use this method going forward.

### CODE REVIEW

**Comment by dj-maisy** [comment]
This might need merging first: https://github.com/alphagov/re-team-manual/pull/229

File: source/documentation/gds-aws-account-management.md [hunk 1]
```diff
@@ -315,3 +315,63 @@ template email useful for this:
 >If you have any further questions, ask on the #reliability-eng slack channel.
 >
 >[Name]
+
+### Closing down AWS accounts
+
+It is a good idea to close down AWS accounts if they are no longer needed.  
+Before you proceed, check that at least one of these scenarios is true:
+
+* The closure request came from a known account owner
```
Comment (jfharden) [inline source/documentation/gds-aws-account-management.md:324]: It's worth making a note in here that the email address used for root on the account can never be used again to open a new aws account in the future.

File: source/documentation/gds-aws-account-management.md [hunk 2]
```diff
@@ -315,3 +315,63 @@ template email useful for this:
...
+ from the `accounts.tf.json` file. If you run `terraform plan` again, you should
+ be able to verify that Terraform will remove/destroy the resources.
+
+When the Pull Request is approved/merged, run the apply command:
+
+```bash
+gds aws billing-org -- sh -c 'terraform apply'
+```
+
+The relevant accounts should now be marked as "suspended" and will be deleted by
```
Comment (jfharden) [inline source/documentation/gds-aws-account-management.md:376]: Also worth linking to this bit of the docs https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/close-account.html#accessing-after-closure that says how you can stop the account from being deleted during the 90 days

File: source/documentation/gds-aws-account-management.md [hunk 3]
```diff
@@ -315,3 +315,68 @@ template email useful for this:
...
+### Closing down AWS accounts
+
+It is a good idea to close down AWS accounts if they are no longer needed.  
+Before you proceed, check that at least one of these scenarios is true:
+
+* The closure request came from a known account owner
+* The account has been abandoned and does not have any active services in 
+production
+* The email address that was used for the root user cannot be reused or 
+ recycled for any other accounts once the account is closed
```
Comment (stephengrier) [inline source/documentation/gds-aws-account-management.md:328]: This doesn't quite read right now. The fact that the root user email address can never be used again is a statement and not a criteria for progressing the account closure. I think just move this down a bit as a separate statement.

**Review by stephengrier — APPROVED** [review]

**Pull request merged**